libs/libcommon/include/common/DateLUTImpl.h

```diff
@@ -14,10 +14,17 @@
 #define DATE_LUT_MAX_YEAR 2105 /// Last supported year
 #define DATE_LUT_YEARS (1 + DATE_LUT_MAX_YEAR - DATE_LUT_MIN_YEAR) /// Number of years in lookup table
 
+/// Flags for calc_week() function.
+#define WEEK_MONDAY_FIRST 1
+#define WEEK_YEAR 2
+#define WEEK_FIRST_WEEKDAY 4
+#define WEEK_NEWYEAR_DAY 8
```

Comment:
I suppose it would be much better to use `enum class WeakMode` (with constants from 0 to 9) here instead of these old-styled macros.

Reply:
ok, I'll fix it.